client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -315,6 +315,10 @@ export default React.createClass( {
 	},
 
 	newDomains() {
+		if ( get( this.props, 'selectedSite.options.is_domain_only' ) ) {
```

Comment:
I'd rather be consistent now and refactor all the changes in one swoop across the file.